src/content/release-notes.mdx

```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
+
+### New Features
```

Comment:
@codyjlandstrom we are adding support for buildkit bridge mode. @jLopezbarb could you create a separate PR to document it in the helm values? And explain the advantages of using bridge mode?

Reply:
PR documenting buildkit network mode:
https://github.com/okteto/docs/pull/819